src/qtiItem/helper/simpleParser.js

```diff
@@ -102,41 +122,68 @@ function buildTooltip(targetHtml, contentId, contentHtml) {
     };
 }
 
+function buildTable($elt, elt, options) {
+    elt.body = {
```

Comment:
It this part needed?
 `body` and `elements` will be copied later

Reply:
yes ,i think it need to be there , unless the assigning of body elements will throw undefined !

Reply:
Object structure needs to be catched before copying , so i think this method is needful